addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -139,4 +139,14 @@
     </xpath>
 </template>
 
+<template id="cart_summary_inherit_website_sale_slides"
```

Comment:
After merge of https://github.com/odoo/odoo/pull/77050 would be moved into a website_sale_templates.xml file (added in that PR :) )

Reply:
Not sure it's the same template, nut I moved it in `website_sale_template` as indeed it makes more sense